source/languages/en/riak/dev/advanced/replication-properties.md

```diff
@@ -92,8 +92,8 @@ Parameter | Common name | Default value | Description
 `n_val` | N | `3` | Replication factor, i.e. the number of nodes in the cluster on which an object is to be stored
 `r` | R | `quorum` | The number of servers that must respond to a read request
 `w` | W | `quorum` | Number of servers that must respond to a write request
-`pr` | PR | `0` | The number of primary <a href="/latest/theory/concepts/vnodes">vnodes</a> that must respond to a read request
-`pw` | PW | `0` | The number of primary <a href="/latest/theory/concepts/glossary#vnode">vnodes</a> that must respond to a write request
+`pr` | PR | `0` | The number of primary <a href="/theory/concepts/vnodes">vnodes</a> that must respond to a read request
+`pw` | PW | `0` | The number of primary <a href="/theory/concepts/glossary#vnode">vnodes</a> that must respond to a write request
```

Comment:
Slight nitpick: this should use the same URL as the previous example, i.e. to [this doc](http://docs.basho.com/riak/latest/theory/concepts/vnodes/). The glossary entry doesn't provide much info.